services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -1099,6 +1102,57 @@ func TestTransformer(t *testing.T) {
 				}
 			},
 		},
+		{
+			Name:       "Lock environment with permissions",
+			DexEnabled: true,
+			Transformers: []Transformer{
+				&CreateEnvironment{Environment: "production"},
+				&CreateEnvironmentLock{
+					Environment: "production",
+					Message:     "don't",
+					LockId:      "manual",
+					RBACConfig: auth.RBACConfig{DexEnabled: true, Policy: map[string]*auth.Permission{
+						"p,developer,EnvironmentLock,Create,production:production,allow": {Role: "Developer"}}},
+				},
+			},
+			Test: func(t *testing.T, s *State) {
```

Comment:
I would prefer to not use a `Test` field here.
Please move your new tests into a new TestFunction that doesn't have that. (maybe call it RbacTransformerTests)

Reply:
Same for `ErrorTest`